docs/metrics/kubernetes-metrics.md

```diff
@@ -6,122 +6,120 @@ description: This page lists the Kubernetes metrics that are collected when you
+Information about filtering and relabeling metrics, and how to send custom Prometheus metrics to Sumo Logic can be found in
+[Collecting Application Metrics guide](https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/main/docs/collecting-application-metrics.md).
 
 ## Kube state metrics (kube-state-metrics)
 
-kube-state-metrics is a service that listens to the Kubernetes API server and generates metrics about the state of the objects, including deployments, nodes, and pods.
+[kube-state-metrics](https://github.com/kubernetes/kube-state-metrics) is a service that listens to the Kubernetes API server and generates metrics about the state of the objects, including deployments, nodes, and pods.
 
-You can deploy kube-state-metrics using Helm. The Sumo Logic helm chart has a dependency to install it. To avoid duplication, use a single instance of FluentD to collect the metrics.  
+You can deploy kube-state-metrics using [The Sumo Logic Kubernetes Collection Helm Chart](https://github.com/SumoLogic/sumologic-kubernetes-collection).
 
 You can use kube-state-metrics in managed Kubernetes environments, such as GKE, EKS, or AKS, or in a Kubernetes deployment you manage yourself.
 
 ### kube-state-metrics metrics to collect
 
-This section lists recommended metrics to collect. For information about all kube-state-metrics, and metric descriptions, see [Kubernetes docs](https://github.com/kubernetes/kube-state-metrics/tree/master/docs)
-
-* Daemonsets
-    * kube_daemonset_status_current_number_scheduled
-    * kube_daemonset_status_desired_number_scheduled
-    * kube_daemonset_status_number_misscheduled
-    * kube_daemonset_status_number_unavailable
-    * kube_daemonset_metadata_generation
-* Deployments
-    * kube_deployment_metadata_generation
-    * kube_deployment_spec_paused
-    * kube_deployment_spec_replicas
-    * kube_deployment_spec_strategy_rollingupdate_max_unavailable
-    * kube_deployment_status_observed_generation
-    * kube_deployment_status_replicas_available
-    * kube_deployment_status_replicas_unavailable
-* Nodes
-    * kube_node_info
-    * kube_node_spec_unschedulable
-    * kube_node_status_allocatable
-    * kube_node_status_capacity
-    * kube_node_status_condition
-* Pods
-    * kube_pod_container_info
-    * kube_pod_container_resource_requests
-    * kube_pod_container_resource_limits
-    * kube_pod_container_status_ready
-    * kube_pod_container_status_terminated_reason
-    * kube_pod_container_status_waiting_reason
-    * kube_pod_status_phase
+This section lists recommended metrics to collect. For information about all kube-state-metrics, and metric descriptions, 
+see [kube-state-metrics docs](https://github.com/kubernetes/kube-state-metrics/tree/master/docs)
```

Comment:
is pointing to `master` safe?

Reply:
I can change the link to point to release branch but we will forget probably to change it https://github.com/kubernetes/kube-state-metrics/tree/release-2.6/docs 🤷

Reply:
I changed the link, we will see in the future if we are able to update it ;)

Reply:
Agree, unfortunately there is no good solution, both of them can end with incorrect information :see_no_evil: